test/command_line/test_search_beam_position.py

```diff
@@ -146,9 +146,9 @@ def test_index_after_search(dials_data, run_in_tmpdir):
 
     # check we can actually index the resulting optimized experiments
     expected_unit_cell = uctbx.unit_cell(
-        (57.780, 57.800, 150.017, 89.991, 89.990, 90.007)
+        (59.663, 64.291, 155.061, 91.376, 89.964, 91.212)
     )
-    expected_rmsds = (0.06, 0.05, 0.001)
+    expected_rmsds = (0.3, 0.5, 0.005)
```

Comment:
Really such a large change in expected RMSD? 🤔

Reply:
Yah, I was surprised too but I'm not sure if there's anything to do about it.